.travis.yml

```diff
@@ -44,15 +46,15 @@ jobs:
             tags: true
```

Comment:
If we're using HubFlow (and my understanding of HubFlow is correct), we need to tweak the deploy harness
such that we deploy on any commit to master. (To accomplish that for this section, we should be able to just remove this line.)

Reply:
We'll do this during the hackathon

Reply:
See https://github.com/ga4gh/data-repository-service-schemas/issues/205